realm/realm-library/src/androidTest/java/io/realm/TypeBasedNotificationsTests.java

```diff
@@ -1401,286 +1398,6 @@ public void onChange() {
         TestHelper.awaitOrFail(signalTestFinished);
     }
 
-    // ****************************************************************************************** //
```

Comment:
They died in the big fire of "refresh() is triggering on next looper event" refactoring